README.md

```diff
@@ -1 +1,45 @@
-# Awesome-ES6
\ No newline at end of file
+# Awesome books with ES6
+
+> This project is about an app of books called Awesome vooks where you can add and remove a bookyou think is awesome
+
+
+## Built Withhttps:
+
+- Html5
+- CSS3
+- Javascript
+- ES6
+
+## Live Demo
+
+## Getting Started
+
+To get a copyof this project you can follow these simple steps.
+
+> open your terminal in your PC
+> type git clone and add this link on the same line https://github.com/Pazzo97/Awesome-ES6
+
```

Comment:
- [ ] As the previous review said, Please write all the proper steps one by one so that user won't get confuse while cloning and running the project in local.

Hint:
```
$ git clone https://github.com/Pazzo97/Awesome-ES6.git
$ cd Awesome-ES6
$ open with vs code and run with live server.
```